code/datums/components/larryknife.dm

```diff
@@ -0,0 +1,43 @@
+//Larry With Knives
+/datum/component/larryknife
```

Comment:
The name could be changes since this component can be applied to any movable and is not exclusive to larry